sematic/function.py

```diff
@@ -39,12 +39,12 @@
 logger = logging.getLogger(__name__)
 
 
-class Calculator(AbstractCalculator):
+class Function(AbstractFunction):
     """
-    A Calculator is Sematic's base unit of computation in a graph.
+    A Function is Sematic's base unit of computation in a graph.
 
-    Functions decorated with the `@calculator` decorator become
```

Comment:
I hadn't even noticed this. First ever iteration?

Reply:
Yes, I had forgotten about it too.